examples/step-33/step-33.cc

```diff
@@ -2109,9 +2109,9 @@ namespace Step33
       case Parameters::Solver::direct:
       {
         SolverControl solver_control (1,0);
-        TrilinosWrappers::SolverDirect direct (solver_control,
-                                               parameters.output ==
-                                               Parameters::Solver::verbose);
+        TrilinosWrappers::SolverDirect::AdditionalData data (
+          parameters.output == Parameters::Solver::verbose);
+        TrilinosWrappers::SolverDirect direct (solver_control, data);
```

Comment:
Why not just create an unnamed temporary object within the call to SolverDirect? It copies the result anyway.

Either way, if you prefer it this way, please merge yourself.